public/html/projects/environment/add.pug

```diff
@@ -9,7 +9,8 @@
 		input.form-control(type="text" ng-model="env.name" placeholder="Friendly name to identify your environment")
 
 		label.control-label Environment
-		textarea.form-control(ng-model="env.json" rows="20" placeholder="This has to be a JSON object")
+		div(ui-ace="{mode: 'json', workerPath: '/public/js/ace/'}" class="form-control" style="height: 200px" ng-model="env.json")
+		//textarea.form-control(ng-model="env.json" rows="20" placeholder="This has to be a JSON object")
```

Comment:
is there any reason to keep this commented out line?

Reply:
ACE editor has no placeholder. I kept this line to preserve placeholder. Perhaps anybody want move this message to label or to tooltip.

Reply:
How about a workaround like this https://stackoverflow.com/questions/26695708/how-can-i-add-placeholder-text-when-the-editor-is-empty

If not can you just add another comment line above explaining that we keep this incase we want to restore the placeholder later. Thanks!